packages/gravity-ui-nunjucks/src/_patterns/03-organisms/00-page-structure/01-page-footer.njk

```diff
@@ -1,7 +1,7 @@
 {%- from 'atoms-icon' import icon -%}
 {%- from 'atoms-image' import image -%}
 
-<footer>
+<footer {% if footer.class %} class="{{ footer.class }}"{% endif %}>
```

Comment:
Unless this is a linting issue you can get rid of one of the spaces here, either inside the `if` or before it.